dist/constants.json

```diff
@@ -8919,1235 +8919,1235 @@
       },
       {
         "value": "644GD",
-        "label": "PAYSON CBOC"
+        "label": "Payson VA Clinic"
       },
       {
         "value": "649GA",
-        "label": "KINGMAN CBOC"
+        "label": "Kingman VA Clinic"
       },
       {
         "value": "649GB",
-        "label": "FLAGSTAFF CBOC"
+        "label": "Flagstaff VA Clinic"
       },
       {
         "value": "649GC",
-        "label": "PRESCOTT CBOC LAKE HAVASU CITY"
+        "label": "Lake Havasu City VA Clinic"
       },
       {
         "value": "649GD",
-        "label": "ANTHEM CBOC"
+        "label": "Anthem VA Clinic"
       },
       {
         "value": "649GE",
-        "label": "COTTONWOOD CBOC"
+        "label": "Cottonwood VA Clinic"
       },
       {
         "value": "649HK",
         "label": "PRESCOTT MORC"
       },
       {
         "value": "678GA",
-        "label": "SIERRA VISTA CBOC"
+        "label": "Sierra Vista VA Clinic"
       },
       {
         "value": "678GB",
-        "label": "YUMA CBOC"
+        "label": "Yuma VA Clinic"
       },
       {
         "value": "678GD",
-        "label": "SAFFORD CBOC"
+        "label": "Safford VA Clinic"
       },
       {
         "value": "678GE",
-        "label": "GREEN VALLEY CBOC"
+        "label": "Green Valley VA Clinic"
       },
       {
         "value": "644GF",
-        "label": "GLOBE HEALTH CARE CLINIC"
+        "label": "Globe VA Clinic"
       },
       {
         "value": "678GG",
-        "label": "SOUTHEAST TUCSON"
+        "label": "Southeast Tucson VA Clinic"
       },
       {
         "value": "678GC",
-        "label": "CASA GRANDE CBOC"
+        "label": "Casa Grande VA Clinic"
       },
       {
         "value": "644GE",
-        "label": "THUNDERBIRD VA CBOC"
+        "label": "Thunderbird VA Clinic"
       },
       {
         "value": "678GF",
-        "label": "NORTHWEST TUCSON URBAN CBOC"
+        "label": "Northwest Tucson VA Clinic"
       },
       {
         "value": "644GG",
-        "label": "N EAST PHOENIX VA CLINIC"
+        "label": "Northeast Phoenix VA Clinic"
       },
       {
         "value": "644GH",
-        "label": "PHOENIX MIDTOWN VA CLINIC"
+        "label": "Phoenix Midtown VA Clinic"
       },
       {
         "value": "678QA",
-        "label": "COCHISE COUNTY VA CLINIC"
+        "label": "Cochise County VA Clinic"
       },
       {
         "value": "644QB",
-        "label": "PHOENIX VA MOBILE CLINIC"
+        "label": "Phoenix VA Mobile Clinic"
       },
       {
         "value": "678QB",
-        "label": "PINAL COUNTY VA CLINIC"
+        "label": "Pinal County VA Clinic"
       },
       {
         "value": "644",
-        "label": "PHOENIX VAMC"
+        "label": "Carl T. Hayden Veterans' Administration Medical Center"
       },
       {
         "value": "649",
-        "label": "NORTHERN ARIZONA HEALTH CARE SYSTEM - PRESCOTT DIVISION"
+        "label": "Bob Stump Department of Veterans Affairs Medical Center"
```

Comment:
- Consider abbreviated label for "Department of Veterans Affairs"
- Let's confirm this name change.

Reply:
Located in Prescott, AZ. So this looks to be the correct facility.